app/assets/stylesheets/main.scss

```diff
@@ -1,3 +1,15 @@
 // Place all the styles related to the main controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+.thumbnail.card {
+  padding: 15px 20px 20px 20px; 
```

Comment:
Shorthand form for property `padding` should be written more concisely as `15px 20px 20px` instead of `15px 20px 20px 20px`